src/StickerService.ts

```diff
@@ -50,6 +52,48 @@ function buildWallpaperCss({
     background-size: cover !important;
   }
 
+  /* Output panel transparency*/
+  .monaco-workbench .part.panel > .content .monaco-editor .monaco-editor-background,
+  div.overflow-guard > div.margin > div.margin-view-overlays, 
+  div.overflow-guard > div.margin
+  {
+    background-color: transparent !important;
+  }
+
+  /* Glass items - common for both */
+  .monaco-list,
+  .monaco-list-rows,
+  .monaco-list-row
+  {
+    background-color: transparent !important;
+  }
+
+  /* Glass notifications */
+  .notification-toast 
+  {
+    backdrop-filter: blur(3px) !important;
+    background-color: rgba(0, 255, 0, 0.4) !important;
```

Comment:
Also these can be in a hex format, they just need the alpha added to them. As this is a script and not a pure CSS file, so I think it should be trivial to create it at runtime by combining it with the colour values from the theme (e.g `"textEditorBackground": "#2f3635",` -> `background-color: #2f363528 !important;

Reply:
I've been procrastinating on refactoring the code to make the theme color's available during asset install.

Looks like I'll have to make those changes if I want to use them sweet new panels :sweat_smile:

Reply:
Pretty much. 
While you can just set everything transparent, it's the difference between [having clear shading](https://user-images.githubusercontent.com/5417183/114447942-7c853f00-9bc2-11eb-8cdb-286ba1d798bb.png) and [losing that while also requiring a *lot* more blurring](https://user-images.githubusercontent.com/5417183/114448022-9292ff80-9bc2-11eb-8992-78efd4ab0ebe.png) to make sense of anything.

Also yea, I'm basically turning everything to glass. Because... why not?

Reply:
Aight fam, we're starting make a bunch of changes and diverging from the existing look and feel. Let's take a step back and just fix:

- [ ] Integrated Terminal Window
- [ ] Output Window
- [ ] Debug Console
- [ ] Glass Pane Notifications

Let's start off with just these contributions. I appreciate the enthusiasm, but let's take it easy.

Reply:
Which is exactly why I'm keeping each change separated as best as I can, so you (and I) can pick and choose. But I'll separate them out to another branch and edit this together to a proper mergeable pull request.

Reply:
Awesome, thanks for being cool! I didn't want to be that guy who takes your pull request, <sup>but then re-writes/removes everything that you added.</sup>

Let me know when you are done with your changes, so I can take a peek again.

Reply:
Hey! Just following up. Was wondering if you where done with your changes. I'll have some time to look at this next weekend.

Reply:
Yup.
The the popup colour/transparency cahnge is for "notifications.background" of the theme JSON, and that's really where it preferably should be done, issue is that then it will also affect users that don't install the wallpaper script.
Here are also the [full glass shenanigans](https://github.com/doki-theme/doki-theme-vscode/compare/master...JohnEdwa:fullglass) if you want to take a look.

And don't worry if you don't want to implement any of it, or you want to modify it in some way, I did all of this for my own use and I'm just presenting it in case you find them useful :)

Reply:
Aight, thanks! I'm currently working on the newest themes, so I'll get to this whenever I'm updating the VSCode plugin (so hopefully later this week). Thanks for getting this bootstrapped!